core/lib/spree/core/controller_helpers/strong_parameters.rb

```diff
@@ -10,6 +10,10 @@ def permitted_attributes
                  to: :permitted_attributes,
                  prefix: :permitted
 
+        def permitted_credit_card_attributes
+          permitted_source_attributes
```

Comment:
@Senjai if that's an issue then I don't think it's specific to this request is it?  Those attributes are already exposed via payments_attributes -> source_attributes on the order (when creating a payment).  Perhaps we need a separate PR for that if it's an issue?